src/app/docs/dropdown/dropdown-docs.component.ts

```diff
@@ -67,4 +67,8 @@ export class DropdownDocsComponent {
       this.demoSettings.style = change.style;
     }
   }
+
+  public actionClicked(name: string): void {
+    console.log('Action clicked:', name);
+  }
```

Comment:
Noticed this slipped in there. Is this deliberate?

Reply:
The demo had a reference to the `actionClicked` callback but the method didn't exist on the class (throwing template check errors). I added the class with a console log so that consumer could see it doing something. I'm happy to remove it or change it if you have something else in mind.

Reply:
Nope, that works. Thanks for taking care of it.